frontend/src/queries/schema/schema-general.ts

```diff
@@ -3103,6 +3114,7 @@ export type InsightFilterProperty =
     | 'stickinessFilter'
     | 'calendarHeatmapFilter'
     | 'lifecycleFilter'
+    | 'properties'
```

Comment:
Is this necessary? Assuming this is done for type compatibility and would prefer an early return for incompatible code paths.